tensorflow/contrib/cmake/tf_core_kernels.cmake

```diff
@@ -36,11 +36,11 @@ if(tensorflow_BUILD_CONTRIB_KERNELS)
       "${tensorflow_source_dir}/tensorflow/contrib/layers/kernels/sparse_feature_cross_kernel.cc"
       "${tensorflow_source_dir}/tensorflow/contrib/layers/ops/bucketization_op.cc"
       "${tensorflow_source_dir}/tensorflow/contrib/layers/ops/sparse_feature_cross_op.cc"
-      "${tensorflow_source_dir}/tensorflow/contrib/rnn/kernels/blas_gemm.cc"
-      "${tensorflow_source_dir}/tensorflow/contrib/rnn/kernels/gru_ops.cc"
-      "${tensorflow_source_dir}/tensorflow/contrib/rnn/kernels/lstm_ops.cc"
-      "${tensorflow_source_dir}/tensorflow/contrib/rnn/ops/gru_ops.cc"
-      "${tensorflow_source_dir}/tensorflow/contrib/rnn/ops/lstm_ops.cc"
+      #"${tensorflow_source_dir}/tensorflow/contrib/rnn/kernels/blas_gemm.cc"
+      #"${tensorflow_source_dir}/tensorflow/contrib/rnn/kernels/gru_ops.cc"
+      #"${tensorflow_source_dir}/tensorflow/contrib/rnn/kernels/lstm_ops.cc"
+      #"${tensorflow_source_dir}/tensorflow/contrib/rnn/ops/gru_ops.cc"
+      #"${tensorflow_source_dir}/tensorflow/contrib/rnn/ops/lstm_ops.cc"
```

Comment:
my bad. I exclude them now by adding them to tf_core_kernels_windows_exclude_srcs